.adr/0010-interest-bearing-channel.md

```diff
@@ -0,0 +1,128 @@
+
+Further: holders of BTC have relatively few options for activating their capital to produce a yield. Assets in EVM environments can generate yield via any number of defi protocols, including Ethereum proof-of-stake (staking pools offer approx %4 yield with near zero overhead).
+
+### Technical
+
+In the dominant use case, PCN payments are brief. The capital lockup time for an individual payment is `O(n)`, where `n` is length of the chain of intermediaries. LN strives toward a mesh network topology so that
+
+- scale requirements can be met (a single hub would struggle to be involved in _all_ payments)
+- privacy claims are defensible (payments are onion routed - each node passes the payment one step further but cannot read the entire route. Would be moot with single hub, and less effective with dense network of hubs)
+
+By contrast, the dominant case in an SCN is for individual capital lockup events (channel fundings) to have non-trivial duration.
+
+For an SCN, the capital lockup time is equal to `fundingTime + channelRunningTime + defundingTime`. Each of `fundingTime` and `defundingTime` is expected to be smaller than LN's `O(n)`, because Nitro makes weaker privacy claims & is happy to disclose the entire intermediary chain at funding time. Despite this, the _total lockup time_ in Nitro will be much bigger on average because of the run-time of funded channels.
+
+## LN Fee mechanism + Commentary
+
+In LN, intermediaries charge a base fee per transaction, plus a scalar fee based on the size of the transaction. In practice, this translates to the base fee acting as a "networking fee" and the scalar acting as a "liquidity rental fee".
+
+Fees on liquidity are required because:
+
+- a given intermediary channel has hard limits on the number of pending transactions it can process at a time (~400)
+- a given intermediary channel has only so much capacity to adjust its balance in one direction or the other (IE, to forward payments)
+
+A channel in a PCN is capacity limited according to its current balance. The limitation is asymmetric: a one BTC channel whose balance is 0.5-0.5 can forward payments up to 0.5 BTC in either direction, but the same channel at a different time with a balance of 0.1-0.9 can forward only 0.1 to the right (and 0.9 to the left). Most LN channels _expect_ to be bi-directional in practice, and for the general flow of payments back and forth to keep the channel balanced on average.
+
+Two potential inefficiencies in this fee structure come to mind:
+
+First, the existing LN fee structure does not specifically account for unbalanced channels. In the case of the 0.1-0.9 channel, the channel operators should much prefer forwarding payments to the left, since those payments re-balance the channel. That preference _could_ be expressed in the market as a discounted fee for leftward payments and a markup for rightward payments, but it is not.
```

Comment:
I think I remember the idea of _negative_ fees to encourage the network to rebalance? In a video you shared...

Reply:
I also remember this. 835c15fd